iosApp/Modules/Screens/Artist/Mocks/MockData.swift

```diff
@@ -14,13 +14,16 @@ class MockArtistViewUIModelProviding: ArtistViewUIModelProviding {
 		ArtistViewUIModel(
 			title: artist.name,
 			headerImageUrl: artist.image,
-			profileImage: URL(string: MockData.url(for: Asset.MockAssets.artist0))!,
+			profileImage: URL(string: "" /*MockData.url(for: Asset.MockAssets.artist0)*/)!,
 			followSection: SupportButton.followButton(),
 			supportSection: SupportButton.supportButton(),
-			trackSection: CellsSectionModel(cells: MockData.bigSongCells_shuffled(seed: 1, onTap: { id in
-				actionHandler.songTapped(id: id)
-			}), title: "LATEST TRACKS"),
-			topSongs: CellsSectionModel(cells: MockData.bigSongCells_shuffled(seed: 2, onTap: { id in actionHandler.songTapped(id: id) }), title: "TOP SONGS"),
+			trackSection: CellsSectionModel(cells: [],
+//											MockData.bigSongCells_shuffled(seed: 1, onTap: { id in
```

Comment:
Same here?

Reply:
I'm leaving all these unused screen modules alone for when/if we bring them back.  I only commented out what was needed to silence compiler errors.  But I removed a lot of these from the build target so they're not being compiled anyway, so yeah I can uncomment it.